c2corg_api/scripts/loadtests/create_test_users.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import logging
+
+from sqlalchemy import engine_from_config
+
+import os
+from sqlalchemy.orm import sessionmaker
+
+from pyramid.paster import get_appsettings
+
+from zope.sqlalchemy import ZopeTransactionExtension
+from c2corg_api.models import Base
+from c2corg_api.scripts.initializedb import setup_db
+from c2corg_api.scripts.migration.sequences import UpdateSequences
+
+from c2corg_api.models.document import DocumentLocale
+from c2corg_api.models.document_history import HistoryMetaData, DocumentVersion
+from c2corg_api.models.user import schema_create_user
+from c2corg_api.models.user_profile import UserProfile
+# from c2corg_api.search.notify_sync import notify_es_syncer
+
+
+# no-op function referenced from `loadtests.ini` (required for
+# `get_appsettings` to work)
+def no_op(global_config, **settings): pass
+
+
+def main(argv=sys.argv):
+    settings_file = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), 'loadtests.ini')
+    settings = get_appsettings(settings_file)
+
+    engine = engine_from_config(settings, 'sqlalchemy.')
+
+    logging.basicConfig()
+    logging.getLogger('sqlalchemy.engine').setLevel(logging.WARN)
+
+    # create a fresh schema on the target database
+    Session = sessionmaker(extension=ZopeTransactionExtension())  # noqa
+    session = Session(bind=engine)
+    Base.metadata.drop_all(engine, checkfirst=True)
```

Comment:
You want to add the test users to a migrated database (containing all documents, users, ... of v5)? In that case you shouldn't use `drop_all`.